crowdsourcing/serializers/project.py

```diff
@@ -66,7 +66,7 @@ def create(self, **kwargs):
                     'module': module.id,
                     'data': json.dumps(row)
                 }
-                task_serializer = TaskSerializer(data=task)
+                task_serializer = TaskSerializer(data=task, fields=('module', 'data'))
```

Comment:
do we need to do bottom up serialization, as in children serializing parent data?